data/campaigns/fri05.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,371 @@
+      .. new_objectives(obj_uplands),
+   h = objective_message_height,
+   position = "topleft",
+}
+
+getting_started_2 = {
+   title = _("Planning the War"),
+   body=reebaud(_("Fortify the chokepoint"),
+      -- TRANSLATORS: Reebaud – Getting Started 2
+      _([[Our most urgent concern, however, should be to secure this pass near our northernmost port. Our enemy must pass through this chokepoint to reach us, but the same is also true in reverse. If we are careful, we can hold the enemy at this point for a long time until we are ready to launch our countercharge. However, the passage is too narrow to be fortified as easily as the Uplands. We must construct as many fortresses tightly together here as possible in order to prevent the enemy from gaining any land. It would be unwise to build anything at all north of the pass, as it would be dangerously exposed.]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Reebaud – Getting Started 2
+      _([[Depending on how many building materials we can spare, it might be wise to demolish some of the other buildings here to make room for even more fortifications. In any case, all civil buildings in this area will be in constant danger of being burnt down during a Danish incursion.]]))
+      .. new_objectives(obj_block),
+   h = objective_message_height,
+   position = "topleft",
+}
+
+getting_started_3 = {
+   title = _("Planning the War"),
+   body=henneke(_("Wood supply"),
+      -- TRANSLATORS: Henneke Lembeck – Getting Started 3
+      _([[As Reebaud points out correctly, the Uplands are the best position for new buildings. Besides their strategic value they offer lots of space for construction, of which the lower island is somewhat short.]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Henneke Lembeck – Getting Started 3
+      _([[As for what we will build, I disagree. More military sites are important, but even more so is the ability to construct buildings in the first place. There are no woodcutters anywhere on the island, and we can’t rely on trade for wood, so we must secure our wood production at once before all our wood has used up by the smokeries and charcoal burners!]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Henneke Lembeck – Getting Started 3
+      _([[The land looks fairly barren, so we will need to reserve more space for foresting than I’m comfortable with. The Lowlands seem to be slightly more fertile, so it might be wise to build the forest infrastructure down here and relocate some of the farms to the Uplands. Farms, reed farms, and berry farms do not depend on fertile land as much as trees do.]]))
```

Comment:
"infrastructure" again

(the last sentence rather emphasises this somewhat unnatural quirk in Widelands :)